modules/performance-lab/module.php

```diff
@@ -0,0 +1,68 @@
+<?php
+namespace Elementor\Modules\PerformanceLab;
+
+use Elementor\Core\Base\Module as BaseModule;
+use Elementor\Plugin;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly
+}
+
+class Module extends BaseModule {
+
+
+	public function get_name() {
+		return 'performance-lab';
+	}
+
+	private function does_performance_lab_is_active() {
+		if ( function_exists( 'webp_uploads_img_tag_update_mime_type' ) ) {
+			$webp_option_name = 'site-health/webp-support';
+			$perflab_modules_settings = get_option( 'perflab_modules_settings' );
+			if ( isset( $perflab_modules_settings ) && isset( $perflab_modules_settings[ $webp_option_name ] ) &&
+							'1' === $perflab_modules_settings[ $webp_option_name ]['enabled'] ) {
+				return true;
+			}
+		}
+		return false;
+	}
+
+	private function performance_lab_get_webp_src( $attachment_id, $size = 'full', $url ) {
+		if ( $this->does_performance_lab_is_active() ) {
```

Comment:
Remove, the filter is set only when it's active
```suggestion
```

Reply:
done